pkg/sync/syncer.go

```
@@ -1440,7 +1437,7 @@
 	if !actionsDone {
 		return nil
 	}
-
+	fmt.Printf("not done: %v\n", graph.Depth)
```

golangci-lint (1.62.2)

use of `fmt.Printf` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$`

(forbidigo)